src/App.jsx

```diff
@@ -1,7 +1,15 @@
-import data from "./data.json";
-
-console.log(data);
+import { Header } from "./components/commons/Header";
+import { Albums } from "./components/Albums";
+import { Footer } from "./components/commons/Footer";
 
 export const App = () => {
-  return <div>Find me in src/app.jsx!</div>;
+  return (
+    <>
+      <Header />
+      <main>
```

Comment:
Maybe the main element could be within the Albums component to keep the App.jsx more clean?